docs/content/en/docs/documentation/access-resources.md

```diff
@@ -0,0 +1,198 @@
+all `Job`s that are referencing it. So we have to efficiently get the list of jobs, and return their ResourceIDs in
+the mapper. So we need an index that maps `Cluster` to `Job`s. Here we can use indexing capabilities of the Informers:
+
+```java
+
+@Override
+public List<EventSource<?, Job>> prepareEventSources(EventSourceContext<Job> context) {
+
+    context.getPrimaryCache()
+            .addIndexer(JOB_CLUSTER_INDEX,
+                    (job -> List.of(indexKey(job.getSpec().getClusterName(), job.getMetadata().getNamespace()))));
+
+    // omitted details
+}
+```
+
+where index key is a String that uniquely identifies a Cluster:
+
+```java
+private String indexKey(String clusterName, String namespace) {
+    return clusterName + "#" + namespace;
+}
+```
+
+In the InformerEventSource for the cluster now we can get all the `Jobs` for the `Cluster` using this index:
+
+```java
+
+InformerEventSource<Job, Cluster> clusterInformer =
+        new InformerEventSource(
+                InformerEventSourceConfiguration.from(Cluster.class, Job.class)
+                        .withSecondaryToPrimaryMapper(
+                                cluster ->
+                                        context.getPrimaryCache()
+                                                .byIndex(
+                                                        JOB_CLUSTER_INDEX,
+                                                        indexKey(
+                                                                cluster.getMetadata().getName(),
+                                                                cluster.getMetadata().getNamespace()))
+                                                .stream()
+                                                .map(ResourceID::fromResource)
+                                                .collect(Collectors.toSet()))
+                        .withNamespacesInheritedFromController().build(), context);
+```
+
+This will trigger all the related `Jobs` if a cluster changes. Also, the maintaining the `PrimaryToSecondaryIndex`.
+So we can use the `getSecondaryResources` in the `Job` reconciler to access the cluster.
+However, there is an issue, what if now there is a new `Job` created? The new job does not propagate
+automatically to `PrimaryToSecondaryIndex` in the `InformerEventSource` of the `Cluster`. That re-indexing
+happens where there is an event received for the `Cluster` and triggers all the `Jobs` again.
+Until that would happen again you could not use `getSecondaryResources` for the new `Job`.
+
+You could access the Cluster directly from cache though in the reconciler:
+
+```java 
+
+@Override
+public UpdateControl<Job> reconcile(Job resource, Context<Job> context) {
+
+    clusterInformer.get(new ResourceID(job.getSpec().getClusterName(), job.getMetadata().getNamespace()));
+
+    // omitted details
+}
+```
+
+But if you still want to use the unified API (thus `context.getSecondaryResources()`), we can add 
```

Comment:
Are there any inconveniences to simply creating a mapper? Why would someone bother with creating an informer index instead? These questions are not answered and the resulting section is rather confused and not really helpful from a user's perspective, imo.